test/anchoredinterval.jl

```diff
@@ -579,14 +579,14 @@ using Intervals: Bounded, Ending, Beginning, canonicalize, isunbounded
         end
 
         @testset "StepRangeLen" begin
-            r = AnchoredInterval{-1}(1):1:AnchoredInterval{-1}(5)
-            r = r[1:5]
+            r = StepRangeLen(AnchoredInterval{-1}(1), 1, 5)
 
             # https://github.com/JuliaLang/julia/issues/33882
             @test r isa StepRangeLen
             @test first(r) == AnchoredInterval{-1}(1)
             @test step(r) == 1
             @test last(r) == AnchoredInterval{-1}(5)
```

Comment:
Should these tests also be run on `AnchoredInterval{-1}(1):1:AnchoredInterval{-1}(5)`?